scripts/tests/ana-collaboration-contract.test.mjs

```
@@ -0,0 +1,202 @@
+test('Ana review routes stay out of search until she approves them', async () => {
+  const [friend, ally, cecilia, download, portal, routeIndexSource, sitemap] = await Promise.all([
+    readRepoFile(pageFiles[0]),
+    readRepoFile(pageFiles[1]),
+    readRepoFile(pageFiles[2]),
+    readRepoFile(pageFiles[3]),
+    readRepoFile('content/portal/ana.ts'),
+    readRepoFile('data/route-index.json'),
+    readRepoFile('app/sitemap.ts'),
+  ])
+
+  assert.match(friend, /noindex: true/)
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)